docs/source/howto_write_config.rst

```diff
@@ -88,14 +89,14 @@ Next we want to preprocess our source code.
 Preprocessing resolves any `#include` and `#ifdef` directives in the code,
 which must happen before we analyse it.
 
-Thanks to Fab's sensible defaults, the Fortran preprocessor know where to find the Fortran source code.
-It was added to the :term:`Artefact Store` by the previous step.
+Thanks to Fab's sensible defaults, the Fortran preprocessor knows where to find the Fortran source code.
+It was added to the :term:`Artefact Store` by the preceding step.
 
 .. note::
     *Artefact Store*
 
-    Steps generally create and find artefacts in this in-memory dict, arranged into named collections.
-    In this case, the Fortran preprocessor automatically looks for source code in a collection named "all_source",
+    Steps generally create and find artefacts in this dict, arranged into named collections.
```

Comment:
In documentation which is not verbatim code I think whole words should be preferred. So "dictionary" in this case.